src/main/scala/com/cognite/sdk/scala/v1/resources/dataPoints.scala

```diff
@@ -562,25 +566,37 @@ object DataPointsResource {
   implicit val aggregateDataPointDecoder: Decoder[AggregateDataPoint] = deriveDecoder
   implicit val aggregateDataPointEncoder: Encoder[AggregateDataPoint] = deriveEncoder
 
-  val asProtobufOrCdpApiError: ResponseAs[
-    Either[DeserializationError[io.circe.Error], Either[CdpApiError, DataPointListResponse]],
-    Nothing
-  ] = {
-    asByteArray.map(response => {
+  private def asProtobufOrError(uri: Uri) =
```

Comment:
Seems like this also parses JSON, and I think that should be reflected in the name.